engine/common/mod_bmodel.c

```diff
@@ -1913,7 +1916,15 @@ static void Mod_LoadVertexes( dbspmodel_t *bmod )
 	{
 		if( bmod->isworld )
 			AddPointToBounds( in->point, world.mins, world.maxs );
+#ifdef XASH_BIG_ENDIAN
```

Comment:
You may completely remove this check. Compiler will optimize out a loop that does nothing on little endian.

Also, if you didn't know, platform macros are defined into positive values in `build.h`, so they can be checked with just `#if` and used in complex expressions without `defined()` usage.